source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -80,674 +80,615 @@ msgstr "IoT agents: agentes para diferentes protocolos."
-"message) does not know which components will receive its message. "
-"Furthermore, any consumer doesn't know who generated the message that it "
-"being ingested. This allows data to be transmitted based on "
-"\"interests\": a consumer is interested in ingesting messages with a "
-"particular `subject` (more on that later) and producers will send "
-"messages to all components that are interested in it. Note that this "
-"mechanism allows multiple services to emit messages with the same "
-"\"subject\", as well as multiple services ingesting messages with the "
-"same \"subject\" with no tricky workarounds whatsoever."
-msgstr ""
-"Kafka, por outro lado, permite uma comunicação pouco acoplada entre "
-"instâncias de serviços. Isso significa que um produtor (quem envia uma "
-"mensagem) não sabe quais componentes receberão sua mensagem. Além disso, "
-"qualquer consumidor não sabe quem gerou a mensagem de que está sendo "
-"ingerido. Isso permite que os dados sejam transmitidos com base em "
-"\"interesses\": um consumidor está interessado em receber mensagens com "
-"um determinado assunto (mais sobre isso mais tarde) e os produtores "
-"enviarão mensagens para todos os componentes que estiverem interessados "
-"nele. Observe que esse mecanismo permite que vários serviços emitam "
-"mensagens com o mesmo \"assunto\", bem como vários serviços que ingerem "
-"mensagens com o mesmo \"assunto\", sem soluções alternativas complicadas."
+"Kafka, in the other hand, allows loosely coupled communication between instances "
+"of services. This means that a producer (whoever sends a message) does not know "
+"which components will receive its message. Furthermore, any consumer doesn't "
+"know who generated the message that it being ingested. This allows data to be "
+"transmitted based on \"interests\": a consumer is interested in ingesting "
+"messages with a particular `subject` (more on that later) and producers will "
+"send messages to all components that are interested in it. Note that this "
+"mechanism allows multiple services to emit messages with the same \"subject\", "
+"as well as multiple services ingesting messages with the same \"subject\" with "
+"no tricky workarounds whatsoever."
+msgstr ""
+"Kafka, por outro lado, permite uma comunicação pouco acoplada entre instâncias "
+"de serviços. Isso significa que um produtor (quem envia uma mensagem) não sabe "
+"quais componentes receberão sua mensagem. Além disso, qualquer consumidor não "
+"sabe quem gerou a mensagem de que está sendo ingerido. Isso permite que os dados "
+"sejam transmitidos com base em \"interesses\": um consumidor está interessado em "
+"receber mensagens com um determinado assunto (*subject*) (mais sobre isso mais tarde) e os "
+"produtores enviarão mensagens para todos os componentes que estiverem "
+"interessados nele. Observe que esse mecanismo permite que vários serviços emitam "
+"mensagens com o mesmo \"assunto\" (*subject*), bem como vários serviços que ingerem "
```

Comment:
> [...] bem como vários serviços que **consomem** [...]

Check if the term in the English version is also suitable!

Reply:
I change in Portuguese, but I think in English "ingesting" is a term used.